config/settings.yml

```diff
@@ -44,6 +50,11 @@
           - security_group.update.end
     :connection_manager:
       :purge_interval: 1.hour
+  :ems_ovirt_network:
```

Comment:
Interesting - once we get rid of the BlacklistedEvent model and all that stuff, I wonder if worker_settings can handle provider subclassing in a way that merges this with the parent. Not required, but something we might want to consider if we want to avoid updating things in 2 places.